test/issues_test.js

```diff
@@ -210,5 +210,21 @@ describe('Issues', function () {
       verifyAll()
       done()
     })
+    it('responds with Bugs given jql query param', function (done) {
+      var req = {
+        query: {
+          jql: 'issueType = Bug'
+        }
+      }
+      var s = [{
+        name: 'foo'
+      }, {
+        name: 'bar'
+      }]
+      _issues.expects('filter').withArgs('issueType = Bug').yields(null, s)
```

Comment:
nothing wrong with mocking other functions IMO, as long as the mocked function is adequately tested :wink: